gabrielaEnache/src/main/java/com/endava/generics/Library.java

```diff
@@ -0,0 +1,70 @@
+package com.endava.generics;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.List;
+
+public class Library {
+    private List<Media> items;
+
+    public Library() {
+        items = new ArrayList<>();
+    }
+
+    public void addItem(Media item) {
+        this.items.add(item);
+    }
+
+    public void addAllItems(List<Media> list) {
+        this.items.addAll(list);
+    }
+
+    public void removeItem(Media item) {
+        this.items.remove(item);
+    }
+
+    public void clearItemList() {
+        this.items.clear();
+    }
+
+    public void removeMultipleItems(List<Media> list) {
```

Comment:
you remove all elements from list, not multiple, for multiple you should have a filter

Reply:
Done